bedrock/settings/base.py

```diff
@@ -294,6 +294,7 @@ def lazy_langs():
     r'^firefox/send-to-device-post',
     r'^firefox/feedback',
     r'^firefox/stub_attribution_code/',
+    r'^.+/tracking-protection/start/$',
```

Comment:
I can't find the reason that this is made for `*/tracking-protection/start/` rather than explicitly for Firefox. Do we expect there to be a page like this for other sections of the site?

Reply:
The URL can be either `/firefox/tracking-protection/start/` or something like `/firefox/58.0/tracking-protection/start/`

Reply:
Ah! Cool. Thanks.